modules/graceful-shutdown.adoc

```diff
@@ -19,6 +24,19 @@ It is important to take an etcd backup before performing this procedure so that
 
 .Procedure
 
+. If you are shutting the cluster down for an extended period, note the date on which certificates expire.
```

Comment:
```suggestion
. If you are shutting the cluster down for an extended period, determine the date on which certificates expire:
```

Reply:
Here, the command is not noting it, but getting the info from the cluster. You could add a followup to this step to say "make a note of this" or similar, but I think the little footnote with the example output actually covers this pretty well already :)